src/CurveLPOracle.sol

```diff
@@ -186,7 +192,7 @@ contract CurveLPOracle {
                 zph_      := shr(24, slot1)
             }
 
-            // When stopped, values are set to zero and should remain such; thus, disallow updating in that case.
+            // When purely stopped or voided, disallow updating in those cases.
```

Comment:
very neat - maybe we should update the comment in the other repos as well (I noted it was not updated at least in https://github.com/makerdao/univ3-lp-oracle/pull/7)